api/src/pcapi/core/reminders/api.py

```diff
@@ -17,12 +17,20 @@ def create_reminder(user: User, reminder: PostReminderRequest) -> FutureOfferRem
     if existing_reminder:
         return existing_reminder
 
-    reminder = FutureOfferReminder(
+    _reminder = FutureOfferReminder(
         userId=user.id,
         futureOfferId=future_offer.id,
     )
 
-    db.session.add(reminder)
+    db.session.add(_reminder)
     db.session.flush()
 
-    return reminder
+    return _reminder
+
+
+def delete_reminder(user: User, reminder_id: int) -> None:
+
+    reminder_query = db.session.query(FutureOfferReminder).filter_by(id=reminder_id, user=user)
+    reminder = reminder_query.first_or_404()
```

Comment:
low logaf, mini suggestion: 
```suggestion
    reminder = db.session.query(FutureOfferReminder).filter_by(id=reminder_id, user=user).first_or_404()
```